src/Integrations/Integrations/CakePHP/CakePHPIntegration.php

```diff
@@ -35,115 +33,16 @@ public function init()
         $integration = $this;
 
         // Since "Dispatcher" and "App" are common names, check for a CakePHP signature before loading
+        /*
         if (!defined('CAKE_CORE_INCLUDE_PATH')) {
             return self::NOT_AVAILABLE;
         }
+        */
```

Comment:
I suppose you will want to continue checking for these for CakePHP V2 (i.e. if the class_exists check fails)?